pkg/tempurl/check_env_linux.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build linux
```

Comment:
Do we need to add `darwin` for macOS？

Reply:
Considering this is only for test purposes, it seems that there is no need to add `darwin `...

Reply:
The source library didn't support darwin quite well in fact.